apis/stash/v1beta1/validator.go

```diff
@@ -1,11 +1,69 @@
 package v1beta1
 
+import "fmt"
+
 // TODO: complete
 func (r BackupSession) IsValid() error {
 	return nil
 }
 
 // TODO: complete
 func (r RestoreSession) IsValid() error {
```

Comment:
We need tests for this.
https://dave.cheney.net/2013/06/09/writing-table-driven-tests-in-go